### ISSUE TITLE
Object Count Widget of Circuit Terminations crashes dashboard.

### ISSUE THREAD

**Issue by PaulWestphal** [netbox-community/netbox#16240]
### Deployment Type

Self-hosted

### NetBox Version

v4.0.2

### Python Version

3.11

### Steps to Reproduce

1. Create a circuitry widget using the UI with Widget Type: Object Counts and Models: Circuits > Circuit Termination.

Here is a JSON of my dashboard:

```

{
    "layout": [
        {
            "h": 5,
            "w": 2,
            "x": 3,
            "y": 0,
            "id": "0ac4a0de-bc58-4a61-be17-5c4ce36a588e"
        },
        {
            "h": 3,
            "w": 8,
            "x": 0,
            "y": 19,
            "id": "5340aaa7-5165-43eb-810b-751cfe45f420"
        },
        {
            "w": 3,
            "x": 8,
            "y": 19,
            "id": "6cba64a9-f416-4c02-b354-1c1620b7ccde"
        },
        {
            "w": 3,
            "x": 9,
            "y": 20,
            "id": "0692cb94-0d2e-4f0a-87c5-0750430dd63a"
        },
        {
            "h": 3,
            "w": 4,
            "x": null,
            "y": null,
            "id": "1198e0b4-be9f-47ca-be21-f158ba2d7470"
        }
    ],
    "config": {
        "0692cb94-0d2e-4f0a-87c5-0750430dd63a": {
            "class": "extras.ObjectListWidget",
            "color": "indigo",
            "title": "Change Log",
            "config": {
                "model": "extras.objectchange",
                "page_size": 100,
                "url_params": null
            }
        },
        "0ac4a0de-bc58-4a61-be17-5c4ce36a588e": {
            "class": "extras.ObjectCountsWidget",
            "color": "pink",
            "title": "IPAM",
            "config": {
                "models": [
                    "ipam.aggregate",
                    "ipam.ipaddress",
                    "ipam.iprange",
                    "ipam.prefix",
                    "ipam.vlan",
                    "ipam.vrf"
                ],
                "filters": null
            }
        },
        "1198e0b4-be9f-47ca-be21-f158ba2d7470": {
            "class": "extras.ObjectCountsWidget",
            "color": "",
            "title": "Object Counts",
            "config": {
                "models": [
                    "circuits.circuittermination"
                ],
                "filters": null
            }
        },
        "5340aaa7-5165-43eb-810b-751cfe45f420": {
            "class": "extras.ObjectCountsWidget",
            "color": "cyan",
            "title": "DCIM",
            "config": {
                "models": [
                    "dcim.cable",
                    "dcim.device",
                    "dcim.devicetype",
                    "dcim.rack",
                    "dcim.site"
                ],
                "filters": null
            }
        },
        "6cba64a9-f416-4c02-b354-1c1620b7ccde": {
            "class": "extras.ObjectCountsWidget",
            "color": "blue",
            "title": "Organization",
            "config": {
                "models": [
                    "dcim.site",
                    "tenancy.contact",
                    "tenancy.tenant"
                ],
                "filters": null
            }
        }
    }
}
```

### Expected Behavior

I expected to still be able to use the netbox dashboard.

### Observed Behavior

I get the following exception whenever I try to access the netbox dashboard now:

```
<class 'django.urls.exceptions.NoReverseMatch'>

Reverse for 'circuittermination_list' not found. 'circuittermination_list' is not a valid view function or pattern name.

Python version: 3.11.6
NetBox version: 4.0.2
Plugins: None installed
```

**Comment by jeremystretch** [comment]
I believe this limitation has been addressed under #15496 and will no longer be an issue in the upcoming v4.0.3 release.

**Comment by jeremystretch** [comment]
Confirmed that this works as expected on the current `develop` branch; will be fixed in v4.0.3.
